Triphouse-project/index.html

```diff
@@ -114,6 +136,38 @@ <h1 class="offer__h1">What do we offer</h1>
                     </div>
                 </div>
             </section>
+            <section class="homes__section">
```

Comment:
![2021-04-26_18h01_26](https://user-images.githubusercontent.com/19935417/116104926-86766a00-a6b9-11eb-9c7b-e65686dee71b.png)
![2021-04-26_18h01_20](https://user-images.githubusercontent.com/19935417/116104929-87a79700-a6b9-11eb-987f-7910df243895.png)
Control was lost on mobile version.

Reply:
Returned the slider in the mobile version